en/messages/common.md

```diff
@@ -1296,643 +1745,6551 @@
    <tbody>
     <tr class="mavlink_field" id="MAV_CMD_NAV_WAYPOINT">
      <td class="mavlink_type" valign="top">16</td>
-     <td class="mavlink_name" valign="top">MAV_CMD_NAV_WAYPOINT</td>
+     <td class="mavlink_name" valign="top">
+      <a href="#MAV_CMD_NAV_WAYPOINT">MAV_CMD_NAV_WAYPOINT</a>

```

Comment:
I think that's exactly what I asked for 👍 .

Reply:
It is. The only thing is that for ENUMS, I use prefix #ENUM_ - ... which you might not like. 

I'm going to merge this. Can you please check out the rendered version just to make sure that you're happy.